src/realm.js

```diff
@@ -175,6 +176,12 @@ export class Realm {
     this.partialEvaluators = (Object.create(null): any);
     this.$GlobalEnv = ((undefined: any): LexicalEnvironment);
 
+    this.react = {
```

Comment:
Do you plan on doing something with this in a follow up? Maybe revert this in the first PR unless it's used?

Reply:
This object is needed, `componentsByName` isn't needed though.